libr/core/rvc.c

```diff
@@ -9,6 +9,24 @@
 #define BPREFIX "branches."
 #define NULLVAL "-"
 
+static char *strip_sys_dir(const char *path) {
```

Comment:
The way this function works is inefficient, a more inefficient way to do it is with strchr but this method wouldn't be as elegent.

Reply:
Ready to merge then?

Reply:
Well I mean maybe it depends on wheter or not you tolarate ugly code or inefficient code